src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.WebAssembly.cs

```diff
@@ -117,7 +117,7 @@ private unsafe int JsIndexOfCore(ReadOnlySpan<char> source, ReadOnlySpan<char> t
             }
         }
 
-        // there are chars that are ignored by ICU hashing algorithm but not ignored by invariant hashing
+        // there are chars that are considered equal by HybridGlobalization but do not have equal hashes when invariant hashed
```

Comment:
"when binary hashed"

Reply:
Does "skipping more characters than ICU would lead to hashes with smaller distribution and more collisions in hash tables" need updating too ?

Reply:
Text about collisions is still true. Only the text in the top does not make sense, it compares ICU hashing result with HG hashing result when it should not.

Reply:
the lower one compares number of collisions, so it's fine